src/main/java/com/notably/logic/commands/HelpCommand.java

```diff
@@ -0,0 +1,22 @@
+package com.notably.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import com.notably.model.Model;
+
+/**
+ * Represent a command that enable/disables the help window.
+ */
+public class HelpCommand extends Command {
+    public static final String COMMAND_WORD = "help";
+
+    @Override
+    public void execute(Model notablyModel) {
+        requireNonNull(notablyModel);
+        if (notablyModel.isHelpOpen()) {
+            notablyModel.setHelpOpen(false);
+        } else {
+            notablyModel.setHelpOpen(true);
+        }
```

Comment:
Actually, shouldn't this command always open the help? I don't think we need the toggling part (if close then open, otherwise close) right?

Reply:
Actually, you're right. Just need to open, no need to toggle.

Reply:
Ahh ok, i wasn't sure if closing is handled by the UI.